app/scripts/modules/amazon/src/function/configure/FunctionTags.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { IWizardPageComponent, MapEditor, FormikFormField, FormValidator } from '@spinnaker/core';
+import { FormikProps } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+import * as _ from 'lodash';
+import { awsTagsValidator } from 'amazon/aws.validators';
+
+export interface IFunctionTagsProps {
+  formik: FormikProps<IAmazonFunctionUpsertCommand>;
+  isNew?: boolean;
+  functionDef: IAmazonFunction;
+}
+
+export class FunctionTags extends React.Component<IFunctionTagsProps>
+  implements IWizardPageComponent<IAmazonFunctionUpsertCommand> {
+  constructor(props: IFunctionTagsProps) {
```

Comment:
omit this empty constructor

Reply:
Done. Change will appear in the next push